app/helpers.php

```diff
@@ -209,20 +209,18 @@ function lazy_load_image($url, $class = '', $alt = '')
 
 function nav_links()
 {
-    $links = [];
+    $links = landing_nav_links();
```

Comment:
actually it should be exactly the same as current menu. The one in flyte's design is just an example.